### PR TITLE
[KOJI-44] Select/Dropdown Input Component

### PR DESCRIPTION
**Pull request by eynglv**
Adds SelectField which uses the existing `TextField`
Screen Shot 2022-10-20 at 2 39 29 PM

**Jira Ticket:** https://gethearth.atlassian.net/browse/KOJI-44

### CODE REVIEW

File: src/components/Field/SelectField/SelectField.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+export type SelectFieldProps = {
+  options: SelectFieldOptions[];
+} & TextFieldProps;
+
+const SelectField = ({
+  options,
+  ...otherProps
+}: SelectFieldProps): JSX.Element => {
+  const [selectedValue, setSelectedValue] = useState('');
```
Comment (eynglv) [inline src/components/Field/SelectField/SelectField.tsx:19]: @smurmus @brianclee-gh i thought about this more... will we be able to leverage the `selectedValue` value if its encapsulated in the hajimari component (as opposed to an izakaya component)

Comment (smurmus) [reply src/components/Field/SelectField/SelectField.tsx:19]: oh hmm ya if we want access to it we will want it to be a level up and pass down ^

File: src/components/Field/SelectField/SelectField.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setSelectedValue(event.target.value);
+  };
+
+  return (
+    <TextField
+      variant='outlined'
+      select
+      id='select'
```
Comment (smurmus) [inline src/components/Field/SelectField/SelectField.tsx:29]: nit: double quotes for props (ugh, really need to set up prettier here properly...)

File: src/components/Field/SelectField/SelectField.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  >
+    {SelectFieldTemplate.bind()}
+  </Story>
+</Canvas>
+
+### Playground
+
+<Canvas>
+  <Story
+    name='Playground'
```
Comment (smurmus) [inline src/components/Field/SelectField/SelectField.stories.mdx:69]: nit: I've moved to calling this `Sandbox`; though maybe `Playground` sounds better...

**Review by smurmus — APPROVED** [review]

**Pull request merged**